examples/lat_lon_lines.py

```diff
@@ -0,0 +1,57 @@
+"""
+============================
+Longitude and Latitude Lines
+============================
+
+How to draw your own (Stonyhurst) longitude and latitude lines
+"""
+import numpy as np
+import astropy.units as u
+import matplotlib.pyplot as plt
+from sunpy.data.sample import AIA_171_IMAGE
+import sunpy.map
+
+###############################################################################
+# We first create the Map using the sample data and import the coordinate
+# functionality.
+from astropy.coordinates import SkyCoord
```

Comment:
Well isn't this more of a walkthrough, so there is no problem with these imports here since its linked to the text.